### PR TITLE
feat: unauthorized homepage view

### PR DESCRIPTION
**Pull request by EvanG13**
### Summary
This PR creates the view for the unauthorized homepage

### Checklist

- [x] Tested the code and it does not break anything
- [x] `npm run format:fix`
- [x] Commits / PR title are easy for others to follow :smiley:
- [x] Include a screenshot if it's for a view :sunglasses:

### CODE REVIEW

**Review by EvanG13 — COMMENTED** [review]
image
Let me know if yall have any opinions. Its pretty simple but I kinda like it that way.

@CleveyP @ahuda620 @CaiminRO

File: package.json [hunk 1]
```diff
@@ -4,6 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "format": "prettier --check .",
+    "format:fix": "prettier --write .",
```
Comment (EvanG13) [inline package.json:8]: I noticed we have prettier installed for the project, so I added two scripts for it.

Comment (EvanG13) [reply package.json]: Moved to a different PR

**Review by CaiminRO — APPROVED** [review]
👍

**Pull request merged**